src/install/dracut-install.c

```diff
@@ -618,7 +618,13 @@ static int resolve_deps(const char *src)
         }
 
         fclose(fptr);
-        wait(&err);
+        int err = -1;
```

Comment:
Shell exit code convention fits. I like it.

Small corner case: if `waitpid()` fails, the fail condition (`err = -1`) will not reflect in `ret`. Not a big matter though.

807c3c3180fe3417295466e0bec99a26a387b048
3f819029ddde17544ebfdeaaa3f8f7e0664cc423
b2d3f8610b3d5a3d385164f5459d0c3938831d11
bb96d3dfe4c74a71b7f5a0070efeb05d6b6eea1b

are ok for me.

Reply:
Hm, I'd expected it to be some form of either WEXITSTATUS() == 0x7f or WTERMSIG() == 0x7f or something but it seems that, indeed, it's neither WIFSIGNALED nor WIFEXITED; a minor rearrangement (and simplification of the return handling) makes this rare (AFAICT corresponding to fork()==-1) case `255`.

Reply:
Moved these to #1845.